src/pyclaw/sharpclaw/solver.py

```diff
@@ -174,13 +177,21 @@ def __init__(self,riemann_solver=None,claw_package=None):
         self._method = None
         self._registers = None
         self.dq_dt = None
-        self.dt_old = None        
+        self.dwdq_dt = None
+        self.dt_old = None
 
         # Used only if time integrator is 'RK'
         self.a = None
         self.b = None
         self.c = None
 
+        # Used only if time integrator is 'DWRK'
+        # self.a, self.at, and self.v are the Shu-Osher arrays
+        self.at = None
```

Comment:
I'd go ahead and call this `a_tilde`.  It would also be nice to add a reference to a paper or book.